gradlew

```diff
@@ -1,5 +1,21 @@
 #!/usr/bin/env sh
 
+#
```

Comment:
i don't think that you should update this file, i not find relationship between your issue and this file

Reply:
All issues you mentioned weirdly I haven't changed manually at all. Maybe my IntelliJ set them automatically, or Gradle did. Wouldn't be able to tell you, but I'll remove/change everything you mentioned 👍